File: Services/Chat/Chat.Application/Feature/Messages/Queries/GetMessage/ResponseMessage.cs
using Chat.Application.Feature.Groups.Queries.GetGroup;
using System;
using System.Collections.Generic;

namespace Chat.Application.Feature.Messages.Queries.GetMessage
{
    public class ResponseMessage
    {
        public DateTime CreatedDate { get; set; } 
        public DateTime? LastModifiedDate { get; set; }
        public bool IsEdited { get; set; }
        public int Id { get; set; }
        public string Message_Id { get; set; }
        public long User_Id { get; set; }
        public long? GroupId { get; set; }
        public long? ToUser_Id { get; set; }
        public long? ReplyToMessageId { get; set; }
        public string Text { get; set; }
        public int? StickerId { get; set; }
        public int? GifId { get; set; }
        public List<FileDto> Files { get; set; }
  
    }
}

File: Apps/bomapp/lib/models/ResponseMessage.dart
class ResponseMessage {
  int? id;
  int? userId;
  int? groupId;
  int? toUserId;
  int? replyToMessageId;
  String? text;
  int? stickerId;
  int? gifId;
  List<String>? files;
  String? messageId;

  ResponseMessage(
      {this.userId,
      this.groupId,
      this.toUserId,
      this.replyToMessageId,
      this.text,
      this.stickerId,
      this.gifId,
      this.files});

  ResponseMessage.fromJson(Map<String, dynamic> json) {
    userId = json['user_Id'];
    groupId = json['groupId'];
    toUserId = json['toUser_Id'];
    replyToMessageId = json['replyToMessageId'];
    text = json['text'];
    stickerId = json['stickerId'];
    gifId = json['gifId'];
    files = json['files'].cast<String>();
    messageId=json['message_Id'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['user_Id'] = this.userId;
    data['group_Id'] = this.groupId;
    data['toUser_Id'] = this.toUserId;
    data['reply_To_MessageId'] = this.replyToMessageId;
    data['text'] = this.text;
    data['sticker_Id'] = this.stickerId;
    data['gif_Id'] = this.gifId;
    data['files'] = this.files;
    return data;
  }
}